.travis.yml

```
@@ -17,9 +17,10 @@ jobs:
       os: osx
 
     - stage: Deploy docker image
-      # Deploy only for pushes to master branch, not other branches, not PRs.
+      # Deploy only for pushes, not PRs, since PRs can't access Travis secrets.
       if: type = push
       script:
+        - if [ -z $TAGS ]; then exit 0; fi  # Skip rest if nothing to deploy.
```

Double quote to prevent globbing and word splitting.
```suggestion
        - if [ -z "$TAGS" ]; then exit 0; fi  # Skip rest if nothing to deploy.
```

---

Rebased and fixed. Thanks.